README.md

```diff
@@ -19,7 +19,14 @@ content as an abstract tree.  It implements the **[unist][unist]** spec.
 ## Introduction
 
 This document defines a format for representing Financial Times article content
-as a tree.  This specification is written in a [typescript][typescript] grammar.
+as a tree. This specification is written in a
+[typescript][https://www.typescriptlang.org/]-like grammar, augmented by the
+addition of the `external` property modifier.
+
+The `external` property modifier indicates that the specified field is absent
+when the `content-tree` is in
+[**transit**](#what-does-it-mean-to-be-in-transit), and required when the
+**content-tree** is at rest.
```

Comment:
this describes the new `external` property modifer. which works like the `readonly` property modifier in typescript, except it only exists in this readme